src/test/java/IntakeTest.java

```diff
@@ -1,44 +0,0 @@
-import static org.junit.jupiter.api.Assertions.*; 
```

Comment:
If your changes break the tests, you need to fix the tests. Won't approve deletion.

Should just be changing the constants, and probably adding two more tests for the cube/cone modes